cookbook/core/type_system/structured_dataset.py

```diff
@@ -87,18 +87,80 @@ def get_subset_df(
 ) -> Annotated[StructuredDataset, subset_cols]:
     df = df.open(pd.DataFrame).all()
     df = pd.concat([df, pd.DataFrame([[30]], columns=["Age"])])
-    # On specifying BigQuery uri for StructuredDataset, Flytekit will write pd.dataframe to a BigQuery table
+    # On specifying BigQuery uri for StructuredDataset, flytekit writes pd.dataframe to a BigQuery table
     return StructuredDataset(dataframe=df)
 
 
 # %%
+# StructuredDataset with ``uri`` Argument
+# ========================================
+#
+# If you specify BigQuery ``uri`` for StructuredDataset, flytekit writes the pandas dataframe to a BigQuery table.
```

Comment:
BigQuery `uri` doesn't mean we're converting pandas DF to BigQuery table. If you open the structured dataset using a different format, then it won't be a pandas DataFrame. Could you just talk about what "uri" is?